internal/storage/storageutil/client.go

```diff
@@ -107,3 +108,11 @@ func CreateTokenSource(storageClientConfig *StorageClientConfig) (tokenSrc oauth
 		return oauth2.StaticTokenSource(&oauth2.Token{}), nil
 	}
 }
+
+// StripScheme strips the scheme from a host if it contains.
+func StripScheme(url string) string {
+	if strings.Contains(url, "://") {
```

Comment:
nit: create const/variable for "://" as it's used at two places in this function.